Vision/Vision.bs

```diff
@@ -86,16 +86,16 @@ Markup Shorthands: markdown yes
 	and its impact will continue to grow in the future, 
 	as it expands reach, knowledge, education, and services even more broadly.
 	We believe the World Wide Web should be 
-	inclusive and respectful of those who use it: 
+	inclusive and respectful of everyone:
 	a Web that supports facts over falsehoods, 
 	people over profits, 
 	humanity over hate. 
 
 # W3C's Vision for the World Wide Web # {#vision-web}
 
 	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all humanity</a>.
-	* The Web is designed for the <a href="https://www.w3.org/TR/ethical-web-principles/#noharm">good of society</a>.
-	* The Web must be <a href="https://www.w3.org/TR/ethical-web-principles/#privacy">safe for those who use it</a>.
+	* The Web is designed for the <a href="https://www.w3.org/TR/ethical-web-principles/#noharm">good of the people who use it</a>.
```

Comment:
I don't personally have a problem with this, but it does +1 the instances of the root "use".

Reply:
On this line, I debated between "good of the people who use it" and just "good of people". I'm okay with either one, really, and open to other alternatives that avoid the problems with "good of society".

Reply:
"Society" has political connotations. "Humanity", "all", "world" … would be preferable ([link to comparison of society synonyms](https://www.thesaurus.com/compare-synonyms?root=society&compare=community,humanity,world&pos=noun&ordinal=1)). It's just that "society" is used a tool to squash non-status quo and forget about those who are not using the web, as @wareid points out. Sadly, society generally references organized society versus indigenous or non-colonial. Is Nick Doty available for review?

Reply:
Again, the Ethical Web principles - from where this term came, and is directly referenced - specifically mentions vulnerable people, so I have to disagree that this could be used as a non-status-quo squashing phrase.  I will also again point out that the change to "society" was done based on suggestion by Sarven, approved by a whole bunch of people.

At this point, my backup suggestion is to directly use the text from the EWP : "The Web does not cause harm to society".

Reply:
I think we want a positive statement rather than a negative one here.

How about
> The Web is designed for the good of all people.
?

Reply:
The Wegb is not designed for the good of all people.  It is, for example, not designed for the good of those who would use it to violate the privacy of those browsing it./

Reply:
It _is_ designed for the good of those people, though -- when they're acting as a user browsing the Web and want their own privacy. And I'm not really sold on the ability to violate others' privacy actually being good for the violator as a person...

(This is why I actually liked the "user" terminology, tbh. It was more clear about these distinctions.)

Reply:
yeah, well, so did I, obvs.  :/